python_modules/dagster/dagster/_cli/api.py

```diff
@@ -673,17 +672,8 @@ def grpc_command(
     ):
         # in the gRPC api CLI we never load more than one module or python file at a time
 
-        module_name = (
-            unwrap_single_code_location_target_cli_arg(kwargs, "module_name")
-            if kwargs["module_name"]
-            else None
-        )
-
-        python_file = (
-            unwrap_single_code_location_target_cli_arg(kwargs, "python_file")
-            if kwargs["python_file"]
-            else None
-        )
+        module_name = kwargs.get("module_name")
```

Comment:
check call would be nice here to document/ensure what type is happening here and why. (this codepath is always called allow_multiple_python_targets is False)

One of the more frustrating parts of working with click is you never know what is in that dictionary so it's nice to put a check call to document and to type it